app/javascript/data.js

```diff
@@ -0,0 +1,207 @@
+import createDebug from 'debug';
+import { onLoad } from './util';
+
+const debug = createDebug('ms:data');
+
+window.store = {};
+window.results = [];
+
+/* globals store, results, ace */
```

Comment:
You can add a module for these by adding it [here](https://github.com/Charcoal-SE/metasmoke/blob/58d6dfc3bdead9ad0489b6f8fd1400fc9db7db11/config/webpack/shared.js#L51).